R/constructors.R

```diff
@@ -249,25 +249,22 @@ cat_quant_param_values <- function(x) {
 
   n_values <- length(values)
 
-  cat_line(glue("Values: {n_values}"))
+  cli::cli_text("Values: {n_values}")
 }
 
 print_transformer <- function(x) {
   if (!is.null(x$trans)) {
-    print(eval(x$trans))
+    text <- utils::capture.output(eval(x$trans))
```

Comment:
i'm using `capture.output()` here because this code uses the print method for transform objects. So we capture it as a string for it to be used in `cli::cli_text()` 

{cli} doesn't like it when you mix and match, so it is better to keep it all {cli}

Reply:
I have switched to `cli_verbatim()` in both places. `cli_text()` ignores `\n` which we need, hence why `cli_verbatim()` was chosen